backend/matching-service/app/.env.development

```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
```

Comment:
ps i didn't realize my template service committed the `.env.development` file, by right not supposed to be committed

I've updated already, can help to update ur service as well